crates/ruff_linter/src/rules/flake8_pyi/rules/unnecessary_literal_union.rs

```diff
@@ -72,6 +72,7 @@ pub(crate) fn unnecessary_literal_union<'a>(checker: &mut Checker, expr: &'a Exp
                     elts,
                     range: _,
                     ctx: _,
+                    is_parenthesized: _,
```

Comment:
What's the benefit of doing this over

```suggestion
                    ..,
```

?

Reply:
It seems the author intended to handle each property explicitly. I didn't want to change that decision.